tutorials/db-migration-spanner-handle-increasing-pks/index.md

```diff
@@ -0,0 +1,363 @@
+However these concepts are out of scope for this tutorial.
+
+1. Similar to previous, import below code for CDC Pipeline and change Primary_IP with mysql’s ip. Then deploy and start the app.
+
+        drop APPLICATION mysql_to_spanner_cdc CASCADE;
+        
+        IMPORT STATIC CustomFunctions.*;
+        
+        CREATE APPLICATION mysql_to_spanner_cdc;
+        
+        CREATE FLOW sample1_cdc;
+        
+        
+        CREATE SOURCE mysql_cdc_source USING MysqlReader  (
+        ConnectionURL: 'jdbc:mysql://PRIMARY_IP:3306/employeedb',
+          Username: 'root',
+          Compression: false,
+          Password_encrypted: 'false',
+          connectionRetryPolicy: 'retryInterval=30, maxRetries=3',
+          FilterTransactionBoundaries: true,
+          Tables: 'employeedb.signin_log',
+          Password: 'password123',
+          SendBeforeImage: true
+          --change this timestamp
+          --,StartTimestamp: '2021-MAR-19 13:00:00'
+          )
+        OUTPUT TO sample_raw_cdc;
+        
+        
+        CREATE STREAM SampleModifiedDataCDC OF Global.WAEvent;
+        
+        CREATE CQ POPULATE_SAMPLE_TABLE_CDC
+        INSERT INTO SampleModifiedDataCDC
+        SELECT * FROM sample_raw_cdc
+        MODIFY(data[0] = bitReverseInt(data[0]));
+        
+        
+        CREATE TARGET spanner_sample_cdc USING Global.SpannerWriter (
+          InstanceID: 'spanner-tgt',
+          --CheckpointTable: 'CHKPOINT',
+          BatchPolicy: 'EventCount: 100, Interval: 10s',
+          ParallelThreads: '',
+          ServiceAccountKey: '/opt/striim/striim-spanner-key.json',
+          Tables: 'employeedb.signin_log,employeedb.signin_log' )
+        INPUT FROM SampleModifiedDataCDC;
+        
+        END FLOW sample1_cdc;
+        
+        END APPLICATION mysql_to_spanner_cdc;
+
+2. Connect to mysql and insert a few rows.
+
+        insert into signin_log (employee_email, details) values ('cdc-test-3@email.com', 'ip 3333 ');
+        insert into signin_log (employee_email, details) values ('cdc-test-4@email.com', 'ip 4444');
+        insert into signin_log (employee_email, details) values ('cdc-test-5@email.com', 'ip 5555');
+
+
+3. Verify data has been replicated into Cloud Spanner.
+
+![spanner cdc complete](7_spanner_cdc_complete.png)
+
+
+## Cleaning up
+
+To avoid incurring charges to your Google Cloud account for the resources used in this tutorial, you can delete the project.
+
+Deleting a project has the following consequences:
+
+- If you used an existing project, you'll also delete any other work that you've done in the project.
+- You can't reuse the project ID of a deleted project. If you created a custom project ID that you plan to use in the
+  future, delete the resources inside the project instead. This ensures that URLs that use the project ID, such as
+  an `appspot.com` URL, remain available.
+
+To delete a project, do the following:
```

Comment:
Instead of recommending to delete the project should we write a few steps on deleting the following resources :

1) Striim instance
2) Cloud SQL 
3) Spanner

Reply:
Deleting project is standard practice for such articles. I would avoid spending too much time on it.